astronomer/providers/snowflake/hooks/snowflake.py

```diff
@@ -65,8 +65,6 @@ def run(  # type: ignore[override]
             with closing(conn.cursor(DictCursor)) as cur:
 
                 for sql_statement in sql:
-
-                    self.log.info("Running statement: %s, parameters: %s", sql_statement, parameters)
```

Comment:
How would anyone know about the progress if we remove this line?

Reply:
can we log only the sql_statement not with params?

Reply:
The logs are coming before the task is deferred when the query is submitted, it makes sense to see how many queries are submitted, let me look at different ways or ask if making keys as secret works fine or not.

Reply:
yes it looks from the conversation that making keys as secret worked, please confirm that point. If yes, we dont need this PR

Reply:
Yes, I have asked, once I get the confirmation will close this.

Reply:
Lets park it for the moment, i am not comfortable with releasing this change until we get clarity

Reply:
Closing this until we get confirmation.